impl/src/settings.rs

```diff
@@ -618,10 +859,9 @@ pub mod tests {
       .ok()
       .expect("Failed to set up temporary directory");
     let cargo_toml_path = temp_workspace_dir.path().join("Cargo.toml");
-    let mut toml = File::create(&cargo_toml_path).unwrap();
-    toml.write_all(toml_contents.as_bytes()).unwrap();
+    std::fs::write(&cargo_toml_path, &toml_contents).unwrap();
 
-    let settings = load_settings(cargo_toml_path).unwrap();
+    let settings = load_settings(cargo_toml_path, None).unwrap();
```

Comment:
Nit: /*name_of_argument=*/None?

Reply:
Sorry, I know you call this out regularly, it's harder for me to catch these things given that vscode renders the names in there: 
<img width="771" alt="Screen Shot 2020-12-14 at 1 16 25 PM" src="https://user-images.githubusercontent.com/26427366/102136758-b0b44300-3e0e-11eb-8c77-97ef396a0268.png">

I'll try to be more conscious of this.

Reply:
woah, that's fancy